Skyscanner/backpack-docs
# [BPK-2744] Improve the lockfile

This changes all Backpack dependencies in the lockfile to be bundled. From what I can tell from reading npm docs, this is what we want. 

I think the file got messed up when people did `npm install` **before cloning submodules**, as it then reverts to the registry or something I guess?

## How to test

Clone this repo, **with submodules**:

```
git clone --recursive git@github.com:Skyscanner/backpack-docs.git
```

Install deps:

```
npm i
```

If all is well, run the site:

```
npm start
```

If everything is 👌you'll see the docs site after a while.
